### ISSUE TITLE
Simultaneous sessions: connection keeps crashing

### ISSUE THREAD

**Issue by aelfinn** [FreeRDP/FreeRDP#6017]
**Describe the bug**
I talked to @antenore about a bug (seemingly in Remmina) that he traced back to FreeRDP. Opening simultaneous sessions in a connection to the same server don’t work—not even via the “Duplicate current connection” button in Remmina. He reported getting a generic error (0x00000005).

Weirdly, in all kinds of tests involving e.g. different amounts of time between logon attempts for the second session, I managed to make it work once but have not been able to reproduce that result.

**To Reproduce**
1. E.g. in Remmina, try to duplicate a session.
2. Logon starts but almost immediately aborts, leaving only the first session open.

**Expected behavior**
A second session should open in which I should be able to logon a second user within the same connection to the RDP server.

**Application details**
Remmina 1.4.1
FreeRDP 1.1.0-beta1

**Desktop (please complete the following information):**
Kubuntu 18.04.4

**Comment by akallabeth** [comment]
@aelfinn 
1. can you reproduce this with `xfreerdp` (this will help narrowing down where there is an issue)
1. Can you reproduce with more recent versions of freerdp?

**Comment by user-72632** [comment]
Not that I have any real useful information here but I was the one who reported this to @antenore a very long time ago! It turned out to be a Remina problem and not FreeRDP; which is why I stopped using it a long time ago because I routinely needed many connections open simultaneously to servers. The version of xfreerdp you are using is very old so I would give a newer one a try. I would be curious to see if newer version of FreeRDP also do it with the current Remina base. If I can did up my old notes I will let you know what I can find.

**Comment by aelfinn** [comment]
>     1. can you reproduce this with `xfreerdp` (this will help narrowing down where there is an issue)

@antenore told me he had already done that and was able to reproduce the unwanted behaviour.
 
>     2. Can you reproduce with more recent versions of freerdp?

I’d love to, but where would I get a suitably recent version? I had thought that I would get it automatically from the Remmina PPA, but it seems not…

**Comment by akallabeth** [comment]
https://github.com/FreeRDP/FreeRDP/wiki/PreBuilds

**Comment by aelfinn** [comment]
Well, maybe I’m doing it wrong. The only way I know of so far to get the version I’m actually using is to type "xfreerdp /version" in a console window. But that still says 1.1.0—that can’t be right, can it?

**Comment by akallabeth** [comment]
@aelfinn oh, you should try the correct binary, it installs to /opt

**Comment by aelfinn** [comment]
Ah, thanks! Is there also a way to make sure Remmina is using that version? (If I don’t want to use the command line.) How would I check that?

**Comment by akallabeth** [comment]
@aelfinn see remmina install instructions for that. Depends which version you're using.

**Comment by user-72632** [comment]
Out of curiosity, I decided to manually install Remmina using the instructions at their [Wiki](https://gitlab.com/Remmina/Remmina/-/wikis/Compilation/Compile-on-Debian-10-Buster) to see if the situation has changed. Therefore, I am using the latest master on both FreeRDP and Remmina and I can say that it appears this has been fixed. I can connect to the same server now whereas before (many years ago) I could not. My situation is slightly different than the OP because I am using Debian 10 (Buster) and I am also using Windows 2012 R2 RDS servers. The OP did not state what Windows version they are connecting to. One thing to check is that you are not ticking the connect to console checkbox. This will give you trouble when trying to connect to multiple sessions if it is not RDS server and a non-admin account. In any event, it looks good now using the latest software. I still could not tell you if it was a Remmina or FreeRDP problem in the past but I do remember that I always could connect to multiple sessions on the same server when I *just* used FreeRDP in a standalone setup or via scripting (i.e. not through Remmina).

HTH
